### PR TITLE
feat: add activity rules section to WorkspaceKind form (#1283)

### PR DESCRIPTION
**Pull request by thaorell**
closes: #1283
 related: #1292 

- Update `swagger.version` file and re-generate Axios client
- Add ActivityRules Modal and ActivityRules table to WorkspaceKind Form under `Workspace Lifecycle and Customization`
- TODO: Add tests
- TODO: Rebase after #1292 is merged
Screenshot 2026-08-04 at 2 20 43 PM

### CODE REVIEW

File: workspaces/frontend/src/app/pages/WorkspaceKinds/Form/activityRules/ActivityRuleModal.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            <ResourceInputWrapper
+              value={String(secondsSinceActive)}
+              type="time"
+              onChange={(value) => setSecondsSinceActive(Number(value))}
+              step={1}
+              aria-label="seconds-since-active"
+            />
+            <HelperText>
+              <HelperTextItem>
+                How long a workspace must run before this rule can apply.. Must be minimum of 15
```
Comment (paulovmr) [inline workspaces/frontend/src/app/pages/WorkspaceKinds/Form/activityRules/ActivityRuleModal.tsx]: The helper text for "Idle Timeout" has three issues: (1) it reads "How long a workspace must run before this rule can apply" which actually describes the "Minimum Running Time" field below, not idle timeout — the idle timeout is about how long since last activity; (2) there's a double period ".."; (3) It says "Must be minimum of 15 seconds" but the submit button is disabled when `secondsSinceActive < 16` (line 181), and the API schema specifies `+kubebuilder:validation:Minimum:=16`. The text should say "16 seconds" to match the actual validation. Consider: "How long a workspace must be idle before this rule takes effect. Must be a minimum of 16 seconds."

File: workspaces/frontend/src/app/pages/WorkspaceKinds/Form/activityRules/ActivityRuleModal.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  return Object.fromEntries(filtered.map((r) => [r.key, r.value]));
+};
+
+export const ActivityRuleModal: React.FC<ActivityRuleModalProps> = ({
+  isOpen,
+  onClose,
+  onSubmit,
+  existingRule,
+}) => {
+  const [secondsSinceActive, setSecondsSinceActive] = useState(15);
```
Comment (paulovmr) [inline workspaces/frontend/src/app/pages/WorkspaceKinds/Form/activityRules/ActivityRuleModal.tsx]: The initial state `useState(15)` for `secondsSinceActive` doesn't match the default from `emptyActivityRule()` which is `3600`. While the `useEffect` overwrites it when the modal opens, the inconsistency is confusing. Consider initializing to `3600` to match the empty rule default, or to `0` to clearly signal "not yet initialized."

File: workspaces/frontend/src/app/pages/WorkspaceKinds/Form/activityRules/WorkspaceKindFormActivityRules.tsx [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+import { LabelGroupWithTooltip } from '~/app/components/LabelGroupWithTooltip';
+
+import { ActivityRuleModal } from './ActivityRuleModal';
+
+interface WorkspaceKindFormActivityRulesProps {
+  activityRules: ActivityRuleEntry[];
+  updateActivityRules: (rules: ActivityRuleEntry[]) => void;
+}
+
+const formatSeconds = (seconds: number): string => {
```
Comment (paulovmr) [inline workspaces/frontend/src/app/pages/WorkspaceKinds/Form/activityRules/WorkspaceKindFormActivityRules.tsx]: `formatSeconds` uses simple division without rounding, so non-round values produce ugly output — e.g., 3661 seconds renders as "1.0169444444444444 hours". Since the API accepts arbitrary integer seconds, consider rounding or displaying the remainder in a smaller unit (e.g., "1 hour 1 minute").

**Review by paulovmr — CHANGES_REQUESTED** [review]
Nice work, @thaorell ! I've added a few comments inline, plus this PR adds Cypress E2E tests (great coverage!) but no Jest unit tests for the new components (`ActivityRuleModal`, `WorkspaceKindFormActivityRules`) or the `formatSeconds` / `labelsToRows` / `rowsToLabels` utilities. There's precedent in the codebase (`WorkspaceKindFormTolerations.spec.tsx`) for unit-testing similar form components. Consider adding unit tests, especially for `formatSeconds` edge cases and the form-to-API data conversion in `convertFormDataToUpdate` for activity rules.

File: workspaces/frontend/src/app/pages/WorkspaceKinds/Form/activityRules/WorkspaceKindFormActivityRules.tsx (hunk 3 above)
Comment (thaorell) [reply workspaces/frontend/src/app/pages/WorkspaceKinds/Form/activityRules/WorkspaceKindFormActivityRules.tsx]: I will be rounding the units to the nearest 0.25 ( 3661s--> 1 hour, 5300s --> 1.5 hour)

**Review by paulovmr — APPROVED** [review]
/lgtm
/approve
